daemon/src/model.rs

```diff
@@ -600,6 +602,73 @@ impl Timestamp {
     }
 }
 
+/// Funding rate per SETTLEMENT_INTERVAL
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct FundingRate(pub Decimal);
+
+impl From<Decimal> for FundingRate {
+    fn from(decimal: Decimal) -> Self {
+        FundingRate(decimal)
+    }
+}
+
+/// Funding parameters used for calculating the fees
+#[derive(Debug, Clone)]
+pub struct FundingParams {
+    funding_rate: FundingRate,
+    /// How much of the funding round are we aiming to pay for now
+    fraction: Decimal,
+    amount_sat: Decimal,
```

Comment:
Can we rename this or at least give it a doc comment. It is not clear from the implementation, what is stored in this value.

Reply:
renamed to 'margin' in the constructor, hopefully it's clear now.